clients/backup/backup.go

```diff
@@ -97,10 +145,12 @@ func (c *Client) downloadMetadata(ctx context.Context, params *Params) error {
 
 	_, contentParams, err := mime.ParseMediaType(rawResp.Header.Get("Content-Type"))
 	if err != nil {
+		rawResp.Body.Close()
 		return err
 	}
 	body, err := api.GunzipIfNeeded(rawResp)
 	if err != nil {
+		rawResp.Body.Close()
 		return err
 	}
 	defer body.Close()
```

Comment:
Can you put this `defer ...` on line ~142 and then remove the `rawResp.Body.Close()`'s on 148 and 153?

Reply:
This defer couldn't move up because it has to be after `body` gets declared, but I changed the `rawResp.Body.Close()` calls to a single `defer` after the request. We might double-close the response but as far as I can tell that's no problem

Reply:
nice!